tools/code-analyzer/src/commands/analyzer/index.ts

```diff
@@ -86,6 +93,8 @@ export default class Analyzer extends Command {
 	 */
 	async run(): Promise< void > {
 		const { args, flags } = await this.parse( Analyzer );
+		const bar = await this.parse( Analyzer );
+		const bex = bar.flags;
```

Comment:
whoopsie, tidy up required 😄

Reply:
Damn, I'm on fire today lol

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/33955/commits/b0e459eaec3146d3b8e374a5305415f2e1efd5a6